examples/pylint/e4702_modified_iterating_dict.py

```diff
@@ -0,0 +1,5 @@
+data = {'x': 1, 'y': 2, 'z': 3}
+
+for key, value in data:
+    data['w'] = 0
+    
```

Comment:
It looks like this line isn't registering as a new blank line at the end of the file.

Could be a newline issue: try changing this file's new lines to macOS/unix style, `\n`. If you're using PyCharm, see https://www.jetbrains.com/help/pycharm/configuring-line-endings-and-line-separators.html.